src/etc/rc.recover

```diff
@@ -62,6 +62,7 @@ audit:*:77:
 www:*:80:
 _ypldap:*:160:
 hast:*:845:
+voucher:*:1003:squid
```

Comment:
@fichtner can you take a look at this? I'm not sure this is the proper way to add a group, next question is, what happens on update...